maximum_height_from_velocity.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""Maximum_height_from_velocity.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1jWWV3KfbMQa5HFUbwPrCsceK0xoWuLcZ
+"""
+
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input, validate_output)
+
+# Description
+## The maximum height to which a body thrown vertically upwards will rise depends on the initial velocity
+## Law: h = (v**2)/(2*g)
+## v is initial velocity,
+## g is acceleration of free fall.
+
+initial_velocity = Symbol("initial_velocity", units.velocity)
+free_fall_acceleration = Symbol("free_fall_acceleration", units.acceleration)
```

Comment:
Free fall acceleration is already present in sympy. You can use it instead your own.